python/plugins/processing/algs/qgis/ServiceAreaFromPoint.py

```diff
@@ -137,8 +138,11 @@ def defineCharacteristics(self):
             p.isAdvanced = True
             self.addParameter(p)
 
-        self.addOutput(OutputVector(self.OUTPUT_LAYER,
-                                    self.tr('Service area'),
+        self.addOutput(OutputVector(self.OUTPUT_POINTS,
```

Comment:
No, Processing does not support optional outputs at the moment. But if you don't specify path or file name it will be saved in the temporary file/memory layer, so I think having both outputs should not hurt.